build/Dockerfile

```diff
@@ -112,6 +112,7 @@ RUN make build-daemon
 #│   ├── csi-proxy.exe
 #├── ecr-credential-provider.exe
 #├── generated/
+#│   └── cri_registry_config/
```

Comment:
Why does this need to be created?

Reply:
ah it doesn't, vestige of the previous solution. I will remove it